core/admin/button.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Elementor\Core\Admin;
+
+
+use Elementor\Utils;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Button extends \Elementor\Core\Base\Module {
+
+	private $options;
+
+	public function __construct( $options ) {
+		$this->options = $options;
+	}
+
+	static public function get_default_options() {
+		return [
+			'class' => 'e-button',
+			'icon' => '',
+			'new_tab' => false,
+			'text' => '',
+			'type' => '',
+			'url' => '',
+			'variant' => '',
+		];
+	}
+
+	private function get_options( $option = false ) {
```

Comment:
And then you can just call:
`return self::get_items( $this->options, $option );`